src/lib.rs

```diff
@@ -509,3 +520,26 @@ impl DerInner<'_> {
         })
     }
 }
+
+// Format an iterator of u8 into a hex string
+fn hex<'a>(f: &mut fmt::Formatter<'_>, payload: impl IntoIterator<Item = &'a u8>) -> fmt::Result {
+    for b in payload {
+        write!(f, "{:02x}", b)?;
+    }
+    Ok(())
+}
+
+#[cfg(feature = "std")]
+mod tests {
```

Comment:
Nit: I would `use super::*` here so we can avoid the `super::` qualifications inside the tests.

Reply:
Fixed.